src/components/Boards.jsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from "react";
+import Board from "./Board";
+import Form from "./Form";
+import { useTrello } from "../context/TrelloContext";
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import { filterBoardById } from "../utils/utils";
+
+const Boards = () => {
+  const [board, setBoard] = useState("");
+  const { boards, setBoards } = useTrello();
+
+  const addTask = (boardId, task) => {
+    // let items = [...boards];
+    // let boardIndex = items.findIndex((item) => item.id === boardId);
+
+    // if (boardIndex < 0) return;
+
+    // let boardItem = boards[boardIndex];
+
```

Comment:
you can remove the commented code if thats moved to some other file

Reply:
done